README.md

```diff
@@ -6,22 +6,67 @@ This project consists of a backend server and a React Native app that demonstrat
 
 * Dialpad for making outgoing calls to a phone number
 * Active call screen to interact with and control ongoing calls
+* Login flow for authenticating and authorizing users
 
 ### Planned Features
 
 * App to app calls
-* Login flow for authenticating and authorizing users
 * Registration and notifications for incoming calls
 
 ![dialer](https://user-images.githubusercontent.com/22135968/223575482-53f733b4-c828-4dd1-b100-7cc945a52c9f.jpg)
 ![active](https://user-images.githubusercontent.com/22135968/223575472-f112ab20-f11c-4f54-9432-3640ddbb0f1c.jpg)
+![login](https://user-images.githubusercontent.com/35968892/226741082-eb0ac83a-b16d-46a3-9ff3-a83167ed7267.png)
 
 ## Prerequisites
 
-- Understanding of [Twilio Programmable Voice](https://www.twilio.com/docs/voice/sdks).
-- A [React Native Development Environment](https://reactnative.dev/docs/0.70/environment-setup).
-- Latest LTS version of [NodeJS](https://nodejs.org/en/).
-- A Node package manager. Examples in this project will use the latest version of [Yarn v1](https://classic.yarnpkg.com/lang/en/).
+* Understanding of [Twilio Programmable Voice](https://www.twilio.com/docs/voice/sdks).
+* A [React Native Development Environment](https://reactnative.dev/docs/0.70/environment-setup).
+* Latest LTS version of [NodeJS](https://nodejs.org/en/).
```

Comment:
Not part of your changes, but I think the official RN docs actually recommend LTS 16 at least for the version of RN we use. There's a `.node-version` file in the project root that is put there by the RN boilerplate, and I've personally had issues with versions of Node that aren't the major version specified in that file. Let's note this as a future task? @charliesantos @kpchoy what do you think?